HerPublicWebsite.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -93,8 +93,8 @@ private enum IncomeThreshold
         { "1525", new LocalAuthorityDetails("Chelmsford City Council", Hug2Status.Pending, "https://www.chelmsford.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
         { "1605", new LocalAuthorityDetails("Cheltenham Borough Council", Hug2Status.Live, "https://www.cheltenham.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
         { "3105", new LocalAuthorityDetails("Cherwell District Council", Hug2Status.Live, "https://www.cherwell.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
-        { "660", new LocalAuthorityDetails("Cheshire East Council", Hug2Status.Pending, "https://www.cheshireeast.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
-        { "665", new LocalAuthorityDetails("Cheshire West and Chester Council", Hug2Status.Pending, "https://www.cheshirewestandchester.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
+        { "660", new LocalAuthorityDetails("Cheshire East Council", Hug2Status.Live, "https://www.cheshireeast.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
+        { "665", new LocalAuthorityDetails("Cheshire West and Chester Council", Hug2Status.Live, "https://www.cheshirewestandchester.gov.uk/", IncomeBandOptions[IncomeThreshold._31000]) },
```

Comment:
Cheshire West and Chester Council isn't mentioned on [PC-615](https://beisdigital.atlassian.net/browse/PC-615). Has it come from somewhere else?

Reply:
Ah, these are consortium members and are mentioned in the row for this specific onboarding task, which can be found in the 'Softwire HUG2' tab of the DSA tracker in column D! It seems the ticket just mentions the main local authority.

Reply:
I see, thanks! Please can you add a comment to the ticket to call these out explicitly? DESNZ are keen that everything is fully captured on Jira.

Reply:
Done, thanks 😄